DependencyInjection/Configuration.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Auto1\ServiceAPIClientBundle\DependencyInjection;
 
+use Psr\Log\LogLevel;
```

Comment:
When you're adding a direct dependency to something - it is a good practice to mention it in composer.
Plz, add `php-fig/log` to composer.json.